docs/api/tutorials/ml.md

```diff
@@ -0,0 +1,500 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# ML System
+
+## Why Would You Integrate ML System with DataHub?
+
+Machine learning systems have become a crucial feature in modern data stacks.
+However, the relationships between the different components of a machine learning system, such as features, models, and feature tables, can be complex.
+Thus, it is essential for these systems to be discoverable to facilitate easy access and utilization by other members of the organization.
+
+For more information on ML entities, please refer to the following docs:
+
+- [MlFeature](/docs/generated/metamodel/entities/mlFeature.md)
+- [MlFeatureTable](/docs/generated/metamodel/entities/mlFeatureTable.md)
+- [MlModel](/docs/generated/metamodel/entities/mlModel.md)
+- [MlModelGroup](/docs/generated/metamodel/entities/mlModelGroup.md)
+
+### Goal Of This Guide
+
+This guide will show you how to
+
+- Create ML entities: MlFeature, MlFeatureTable, MlModel, MlModelGroup
+- Read ML entities: MlFeature, MlFeatureTable, MlModel, MlModelGroup
+- Attach MlFeatureTable or MlModel to MlFeature
+
+## Prerequisites
+
+For this tutorial, you need to deploy DataHub Quickstart and ingest sample data.
+For detailed steps, please refer to [Datahub Quickstart Guide](/docs/quickstart.md).
+
+## Create ML Entities
+
+### Create MlFeature
+
+<Tabs>
+<TabItem value="python" label="Python" default>
+
+```python
+{{ inline /metadata-ingestion/examples/library/create_mlfeature.py show_path_as_comment }}
+```
+
+Note that when creating a feature, you can access a list of data sources using `sources`.
+
+</TabItem>
+</Tabs>
+
+### Create MlFeatureTable
+
+<Tabs>
+<TabItem value="python" label="Python" default>
+
+```python
+{{ inline /metadata-ingestion/examples/library/create_mlfeature_table.py show_path_as_comment }}
+```
+
+Note that when creating a feature table, you can access a list of features using `mlFeatures`.
+
+</TabItem>
+</Tabs>
+
+### Create MlModel
+
+<Tabs>
+<TabItem value="python" label="Python" default>
+
+```python
+{{ inline /metadata-ingestion/examples/library/create_mlmodel.py show_path_as_comment }}
+```
+
+Note that when creating a model, you can access a list of features using `mlFeatures`.
+Additionally, you can access the relationship to model groups with `groups`.
+
+</TabItem>
+</Tabs>
+
+### Create MlModelGroup
```

Comment:
MlModelGroup is a container for all the runs of a single ML Model, so some users might conceptualize this as a ML Model